test/pkg/environment/common/expectations.go

```diff
@@ -294,6 +294,19 @@ func (env *Environment) EventuallyExpectTerminatingWithTimeout(timeout time.Dura
 	}).WithTimeout(timeout).Should(Succeed())
 }
 
+func (env *Environment) eventuallyExpectNoLeakedKubeNodeLease() {
```

Comment:
```suggestion
func (env *Environment) EventuallyExpectNoLeakedKubeNodeLease() {
```

Why make this private?